data/configs/cycle_hcr_probe_designer.yaml

```diff
@@ -78,7 +78,7 @@ target_probe_secondary_structures_threshold_deltaG: 0 # threshold for the second
 # Specificity filter with BlastN
 target_probe_specificity_blastn_search_parameters:
   -perc_identity: 100
-  -strand: "plus" # this parameter is fixed, if reference is whole genome, consider using "both"
+  -strand: "minus" # this parameter is fixed, if reference is whole genome, consider using "both"
```

Comment:
why this change? do we need to document it in a NEWS.md?

Reply:
technically we don't change anything with this change, as we apply now the search to the oligo instaed of the target, i.e. the results stay the same